pkg/restapi/rp/operation/credentials.go

```diff
@@ -17,76 +17,51 @@ import (
 	"github.com/trustbloc/edge-adapter/pkg/internal/common/adapterutil"
 	"github.com/trustbloc/edge-adapter/pkg/presexch"
 	"github.com/trustbloc/edge-adapter/pkg/vc"
-	"github.com/trustbloc/edge-adapter/pkg/vc/rp"
 )
 
 var errInvalidCredential = errors.New("malformed credential")
 
-func parseWalletResponse(definitions *presexch.PresentationDefinitions, vdriReg vdriapi.Registry,
-	vpBytes []byte) (*vc.AuthorizationCredential, *verifiable.Credential, error) {
+func parseWalletResponse(definitions *presexch.PresentationDefinitions, _ vdriapi.Registry,
+	vpBytes []byte) (local, remote map[string]*verifiable.Credential, err error) {
```

Comment:
Can we remove the unused param `_ vdriapi.Registry` ?

Reply:
@rolsonquadras removed

Reply:
although I have a feeling I'll have to add it again when I test integration with the sandbox... I'll update then (if needed)